pkg/util/kubernetes/kubernetes.go

```diff
@@ -114,8 +114,7 @@ func CreateMgmtService(kubecli corev1client.CoreV1Interface, clusterName, cluste
 			Name:       "monitoring",
 			Port:       constants.MonitoringPort,
 			TargetPort: intstr.FromInt(constants.MonitoringPort),
-			Protocol:   v1.ProtocolTCP,
-		},
+			Protocol:   v1.ProtocolTCP,		},
```

Comment:
This formatting was made by gofmt or yourself? Nothing special but a little weird.

Reply:
ah interesting, looks like it does that when there is no trailing comma

Reply:
looks like didn't do gofmt well here, addressed in other commit